include/fx/gltf.h

```diff
@@ -978,12 +979,14 @@ namespace gltf
                     throw std::system_error(std::make_error_code(std::errc::io_error));
                 }
 
-                const char empty[3] = {};
                 fileData.write(reinterpret_cast<char *>(&header), detail::HeaderSize);
                 fileData.write(jsonText.c_str(), jsonText.length());
+                const char spaces[3] = {' ', ' ', ' '};

```

Comment:
I'd prefer to have both spaces and nulls defined about the block of code doing all the writes - can you just shift their declarations to where empty used to be?  Mind the spacing here too : { blah, blah, blah };

Reply:
Thanks for the feedback / recommendations, I'll patch that up for us both.